docs/diagnostics/overview.md

```diff
@@ -2,97 +2,205 @@
 title: .NET Aspire diagnostics overview
 description: Learn about the diagnostics tools and features available in .NET Aspire.
 ms.topic: overview
-ms.date: 10/21/2024
+ms.date: 04/15/2025
+ai-usage: ai-assisted
 ---
 
 # .NET Aspire diagnostics overview
 
-Several APIs of .NET Aspire are decorated with the <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>. This attribute indicates that the API is experimental and may be removed or changed in future versions of .NET Aspire. The attribute is used to identify APIs that aren't yet stable and may not be suitable for production use.
+Several APIs of .NET Aspire are decorated with the <xref:System.Diagnostics.CodeAnalysis.ExperimentalAttribute>. This attribute indicates that the API is experimental and might be removed or changed in future versions of .NET Aspire. The attribute is used to identify APIs that aren't yet stable and might not be suitable for production use.
 
-## AZPROVISION001
+## ASPIREACADOMAINS001
+
+<span id="ASPIREACADOMAINS001"></span>
 
-<span id="AZPROVISION001"></span>
+.NET Aspire 9.0 introduces the ability to customize container app resources using any of the following extension methods:
 
-.NET Aspire provides various overloads for Azure Provisioning resource types (from the `Azure.Provisioning` package). The overloads are used to create resources with different configurations. The overloads are experimental and may be removed or changed in future versions of .NET Aspire.
+- <xref:Aspire.Hosting.AzureContainerAppProjectExtensions.PublishAsAzureContainerApp*>
+- <xref:Aspire.Hosting.AzureContainerAppContainerExtensions.PublishAsAzureContainerApp*>
+- <xref:Aspire.Hosting.AzureContainerAppExecutableExtensions.PublishAsAzureContainerApp*>
```

Comment:
This should make the text different for each item, otherwise it just looks like 3 links that are exactly the same 😁 

```suggestion
- <xref:Aspire.Hosting.AzureContainerAppProjectExtensions.PublishAsAzureContainerApp*?displayProperty=nameWithType>
- <xref:Aspire.Hosting.AzureContainerAppContainerExtensions.PublishAsAzureContainerApp*?displayProperty=nameWithType>
- <xref:Aspire.Hosting.AzureContainerAppExecutableExtensions.PublishAsAzureContainerApp*?displayProperty=nameWithType>
```